src/Monitoring.cxx

```diff
@@ -83,9 +82,12 @@ void Monitoring::flushBuffer(const short index)
   mStorage[index].clear();
 }
 
-void Monitoring::enableProcessMonitoring(const unsigned int interval)
+void Monitoring::enableProcessMonitoring(const unsigned int interval, std::vector<Monitor> enabledMeasurements)
```

Comment:
@ktf Are you okay with that?
For analysis you'll need to call: `enableProcessMonitoring(15, {Monitor::Cpu, Monitor::Mem, Monitor::Smaps})`.
For anything else `enableProcessMonitoring(15)` would be enough (as `Cpu` and `Mem` are default).

Reply:
Fine by me. I will have in any case to adapt.